src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -83,32 +125,33 @@ You can combine selectors using boolean logic.
 
 ### Value Types
 
-Select subsections by JSON-type or semantic meaning using the following:
-
-* `$string`
-* `$number`
-* `$boolean`
-* `$datetime`
-* `$array`
-* `$object`
-* `$event`
-* `$exception`
-* `$stacktrace`
-* `$frame`
-* `$request`
-* `$user`
-* `$logentry` (also applies to `event.message`)
-* `$thread`
-* `$breadcrumb`
-* `$span`
-* `$sdk`
-
-Examples:
+Select subsections by JSON-type using the following:
+
+* `$string` matches any string value
+* `$number` matches any integer or float value
+* `$datetime` matches any field in the event that represents a timestamp
+* `$array` matches any JSON array value
+* `$object` matches any JSON object
+
+Select known parts of the schema using the following:
+
+* `$exception` matches a single exception instance in `{"exception": {"values": [...]}}`
+* `$stacktrace` matches a stack trace instance
+* `$frame` matches a frame
+* `$request` matches the HTTP request context of an event
+* `$user` matches the user context of an event
+* `$logentry` (also applies to the `message` attribute)
```

Comment:
This line doesn't match the format used in the other lines around it. Suggestion:

```suggestion
* `$logentry` matches both the `logentry` attribute of an event as well as the `message` attribute
```

By the way, looking at the [source code in Relay](https://github.com/getsentry/relay/blob/31593da8c59876434406c6dd00007b9b71674847/relay-general/src/protocol/event.rs#L236-L239) I have the impression that `message` is an alias to `logentry`.

Perhaps we shall not document it here? That was the decision when I was updating dev docs on interfaces.